MAESTRO/Snakemake/scATAC/Snakefile

```diff
@@ -52,42 +52,23 @@ if config["platform"] == "microfluidic":
             fastq1 = os.path.join(config["fastqdir"], "{fastqid}_1.fastq"),
             fastq2 = os.path.join(config["fastqdir"], "{fastqid}_2.fastq")
         output:
-            bam = temp("Result/minimap2/{fastqid}.sortedByPos.bam")
+            bam = "Result/minimap2/{fastqid}.bam"
         threads:
             config["cores"]
         shell:
             "minimap2 -ax sr -t {threads} {input.fasta} {input.fastq1} {input.fastq2} "
             "| samtools view --threads {threads} -b "
             "| samtools sort --threads {threads} -o {output.bam}"
 
-    rule scatac_bamrmdp:
```

Comment:
for microfluidic data, we align fastq per cell to get a bam file. We probably want to merge the bams for all cells first, and then Mark the duplicates for the merged bam and name it as `{sample}.sortedByPos.mkdp.CBadded.bam`.

Similarly for 10x data, let's still do mark the duplicates and name it as `{sample}.sortedByPos.mkdp.CBadded.bam` rather than `{sample}.sortedByPos.rmdp.CBadded.bam`.

We still want to samtools flagstat {sample}.sortedByPos.mkdp.CBadded.bam to get the duplicates rates at bulk level. I see you removed this step in the `scatac_bulk_qcstat`